infinitory/inventory.py

```diff
@@ -2,30 +2,35 @@
 from operator import itemgetter
 from simplepup import puppetdb
 
+import infinitory.errors as errors
+
+
 class Inventory(object):
     def __init__(self, filters=set()):
         self.filter = puppetdb.QueryFilter(filters)
         self.nodes = None
         self.roles = None
 
+        self.errorParser = errors.ErrorParser()
+
     def add_active_filter(self):
         self.filter.add("nodes { deactivated is null and expired is null }")
 
     def add_filter(self, filter):
         self.filter.add(filter)
 
-    def load_nodes(self, pdb):
```

Comment:
Why change all of these to `pupdb`? `pdb` is a more standard abbreviation, and it makes this PR noisier than it needs to be.

Reply:
because pdb is the name of the "Python Debugger".  And it's conflicting every time I try to use it:

https://docs.python.org/3/library/pdb.html